lldb/test/API/lang/swift/variables/move_function_async/TestSwiftMoveFunctionAsync.py

```diff
@@ -0,0 +1,171 @@
+# TestSwiftMoveFunctionAsync.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Check that we properly show variables at various points of the CFG while
+stepping with the move function.
+"""
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import sys
+import unittest2
+
+def stderr_print(line):
+    sys.stderr.write(line + "\n")
+
+class TestSwiftMoveFunctionAsyncType(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    @swiftTest
+    def test_swift_move_function_async(self):
+        """Check that we properly show variables at various points of the CFG while
+        stepping with the move function.
+        """
+        self.build()
+
+        self.exec_artifact = self.getBuildArtifact(self.exec_name)
+        self.target = self.dbg.CreateTarget(self.exec_artifact)
+        self.assertTrue(self.target, VALID_TARGET)
+
+        self.do_setup_breakpoints()
+
+        self.process = self.target.LaunchSimple(None, None, os.getcwd())
+        threads = lldbutil.get_threads_stopped_at_breakpoint(
```

Comment:
Can you replace this boilerplate with a call to 
```
        target, process, thread, bkpt = lldbutil.run_to_source_breakpoint(
            self, 'Set breakpoint', lldb.SBFileSpec('main.swift'))
```
? It doesn't look like the numbering of the breakpoints is actually used, so why not simplify it.